bundles/org.openhab.binding.deutschebahn/README.md

```diff
@@ -0,0 +1,345 @@
+the [Timetables V1 API Description](https://developer.deutschebahn.com/store/apis/info?name=Timetables&version=v1&provider=DBOpenData&).
+The informations are grouped into three channel-groups:
+The first channel group (trip) contains all informations for the trip of the train, for example the category (like ICE, RE, S).
+The second and third channel group contains informations about the the arrival and the departure of the train at the given station. 
+Both of the groups may provide an 'UNDEF' channel value, when the train does not arrive / depart at this station
+(due it starts or ends at the given station). If you have configured your timetable to contain only departures (with property trainFilter) the departure channel values will always be defined 
+and if you have selected only arrivals the arrival channel values will always be defined.
+Channels will have an 'NULL' channel value, when corresponding attribute is not set.
+
+Basically most informations are available as planned and changed value. This allows to easy display changed values (for example the delay or changed platform).
+
+
+**Channels for trip information**
+| channel  | type   | description                  |
+|----------|--------|------------------------------|
+| category | String | Provides the category of the trip, e.g. "ICE" or "RE". |
+| number | String | Provides the trip/train number, e.g. "4523". |
+| filter-flags | String | Provides the filter flags. |
+| trip-type | String | Provides the type of the trip. |
+| owner | String | Provides the owner of the train. A unique short-form and only intended to map a trip to specific evu (EisenbahnVerkehrsUnternehmen). |
+
+
+**Channels for arrival / departure**
+| channel  | type   | description                  |
+|----------|--------|------------------------------|
+| planned-path  | String | Provides the planned path of a train. |
+| changed-path  | String | Provides the changed path of a train. |
+| planned-platform  | String | Provides the planned platform of a train.  |
+| changed-platform | String | Provides the changed platform of a train. |
+| planned-time | DateTime | Provides the planned time of a train. |
+| changed-time | DateTime | Provides the changed time of a train. |
+| planned-status | String | Provides the planned status (planned, added, cancelled) of a train. |
+| changed-status | String | Provides the changed status (planned, added, cancelled) of a train. |
+| cancellation-time | DateTime | Time when the cancellation of this stop was created. |
+| line | String | The line of the train.  |
+| messages | String | Messages for this train. Contains all translated codes from the messages of the selected train stop. Multiple messages will be separated with an single dash. |
+| hidden | Switch | On if the event should not be shown on WBT because travellers are not supposed to enter or exit the train at this stop. |
```

Comment:
WBT?

Reply:
The descriptions are taken from the original api documentation, so even i don't know what WBT means. I think it might be "Wall Board Timetable" or something else. So due this has no relevance here, i removed it.